packages/playground/client/src/index.ts

```diff
@@ -100,24 +153,18 @@ export async function connectPlayground(
 	iframe: HTMLIFrameElement,
 	options?: { loadRemote?: string }
 ): Promise<PlaygroundClient> {
+	console.warn(
+		'`connectPlayground` is deprecated and will be removed. Use `startPlayground` instead.'
+	);
```

Comment:
nice: deprecating with warning vs. simply removing it

Reply:
Thanks! I think there's one or two apps out there relying on this function and even though there are warnings all over the place, I'd still like to give them a grace period.